pkgs/tools/security/solo2-cli/default.nix

```diff
@@ -32,6 +32,8 @@ rustPlatform.buildRustPackage rec {
     ++ lib.optionals stdenv.isDarwin [ PCSC IOKit CoreFoundation AppKit ];
 
   postInstall = ''
+    mkdir -p $out/lib/udev/rules.d
+    cp 70-solo2.rules $out/lib/udev/rules.d/
```

Comment:
`install -D 70-solo2.rules $out/lib/udev/rules.d/70-solo2.rules`

maybe (and skip the mkdir)?

Reply:
Done, thanks!